FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -110,6 +121,31 @@ trY = 2110.326736121985;
 
 // Maslow main loop, everything is processed here
 void Maslow_::update(){
+  
+    if(random(10000) == 0){
+        digitalWrite(ETHERNETLEDPIN, LOW);
+    }
+    
+    if(random(10000) == 0){
+        digitalWrite(ETHERNETLEDPIN, HIGH);
+    }
+
+    if(random(10000) == 0){
+        digitalWrite(WIFILED, LOW);
+    }
+
+    if(random(10000) == 0){
+        digitalWrite(WIFILED, HIGH);
+    }
+
+    if(random(10000) == 0){
+        digitalWrite(REDLED, LOW);
+    }
+
+    if(random(10000) == 0){
+        digitalWrite(REDLED, HIGH);
+    }
+  
```

Comment:
I'd move into separate function

Reply:
This is really just a test. We need to decide what the LEDs should do and make a separate function which does things with them. Maybe the blue one should blink out the IP address when connected to wifi, or the orange ones around the ethernet ports should do the fancy "breathe" thing where they get brighter and dimmer gradually? What are your thoughts?